File: Lopdrinks-main/server/app/constants/order_status.py
"""Order lifecycle status constants."""


class OrderStatus:
    PENDING: str = "Pending"
    CONFIRMED: str = "Confirmed"
    SHIPPED: str = "Shipped"
    DELIVERED: str = "Delivered"
    CANCELLED: str = "Cancelled"

    ALL: tuple[str, ...] = (PENDING, CONFIRMED, SHIPPED, DELIVERED, CANCELLED)

    # Only pending orders may be cancelled/deleted by the owning user
    USER_CANCELLABLE: tuple[str, ...] = (PENDING,)

File: Lopdrinks-main/main/src/constants/orderStatus.js
/** All valid order status values returned by the API. */
export const ORDER_STATUS = {
  PENDING: 'Pending',
  CONFIRMED: 'Confirmed',
  SHIPPED: 'Shipped',
  DELIVERED: 'Delivered',
  CANCELLED: 'Cancelled',
};

export const ORDER_STATUS_OPTIONS = Object.values(ORDER_STATUS);
